src/content/en/updates/2020/06/devtools.md

```diff
@@ -0,0 +1,215 @@
+
+![Bytecode offsets](/web/updates/images/2020/06/bytecode-offset.png)
+
+Chromium issue [#1071432](https://crbug.com/1071432)
+
+## Line-wise copy and cut in Sources Panel {: #sources-panel }
+
+[sources]: https://developers.google.com/web/tools/chrome-devtools/sources#edit
+
+When performing copy or cut with no selection in the [Sources panel editor][sources], DevTools
+will copy or cut the current line content. For example, in the video below,
+the cursor is at the end of line 1. After pressing the cut keyboard shortcut, the
+entire line is copied to the clipboard and deleted.
+
+<video autoplay loop muted playsinline>
+  <source src="/web/updates/images/2020/06/line-wise-cut.mp4" type="video/mp4">
+</video>
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend.git/+/05d110464566dabd3b90e8123d3d2e33aebc9434 #}
+
+Chromium issue [#800028](https://crbug.com/800028)
+
+
+## Console Settings updates {: #console-settings }
+
+### Ungroup same console messages {: #ungroup-messages }
+
+The **Group similar** toggle in Console Settings now applies to duplicate messages.
+Previously it just applied to similar messages.
+
+For example, previously, DevTools did not ungroup the messages `hello` even though
+**Group similar** is unchecked. Now, the `hello` messages are ungrouped:
+
+<video autoplay loop muted playsinline>
+  <source src="/web/updates/images/2020/06/ungroup-similar.mp4" type="video/mp4">
+</video>
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/c84588184b0074cd229ae4dc496e2f11b92dc340 #}
+
+Chromium issue [#1082963](https://crbug.com/1082963)
+
+### Persisting **Selected context only** settings {: #maskable-icons }
+
+The **Selected context only** settings in Console Settings is now persisted.
+Previously the settings were reset every time you closed and reopened DevTools.
+This change makes the setting behavior consistent with other Console Settings options.
+
+![Selected context only](/web/updates/images/2020/06/selected-context.png)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/0c3989432b058d853ab9183354a93bca73427f12%5E%21/#F0 #}
+
+Chromium issue [#1055875](https://crbug.com/1055875)
+
+## Performance panel updates {: #perf-panel }
+
+### JavaScript compilation cache information in Performance panel {:js-compilation-cache}
+[JavaScript compilation cache information](https://v8.dev/blog/code-caching-for-devs) information is now always displayed in the Summary tab of the Performance panel. Previously, DevTools wouldn’t show anything related to code caching if code caching didn’t happen.
+
+![JavaScript compilation cache information](/web/updates/images/2020/06/js-compilation-cache.png)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/bff241dc95433dccd0621c81d93aae793d46a48e #}
+
+Chromium issue [#912581](https://crbug.com/912581)
+
+### Align navigation timing in Performance panel {:nav-timing}
```

Comment:
```suggestion
### Navigation timing alignment in the Performance panel {:nav-timing}
```
@chybie I'm rephrasing these because "align navigation timing" sounds like an action that the user takes, whereas the more passive phrasing of "Navigation timing alignment" makes it clearer that this is a change that happened in DevTools (that doesn't require any action by the user)